lib/tasks/translate_collections.rake

```diff
@@ -0,0 +1,277 @@
+require 'libxml_to_hash'
```

Comment:
General comment for this file: there's a lot of custom code involving the manipulation of data structures without a lot of comments.  In particular, comments about the expected shapes of the inputs and outputs would be helpful throughout.

Reply:
Will do! Thanks for parsing through all of this so carefully - I'm taking note of each comment for my future reference